### DISCUSSION TITLE
Exiting due to prerender errors.

### DISCUSSION THREAD

**Discussion by heiyehk** [nuxt/content#2468] [Q&A]
nuxt3 content
I used [...slug].vue when using generate, and then used ContentDoc in the code, but encountered an error when generating, like the following
![image](https://github.com/nuxt/content/assets/33891067/df3fc16b-3799-4d28-98ef-1f5698d0505c)

But it works fine when I comment out ContentDoc. Do I need to do anything?
![image](https://github.com/nuxt/content/assets/33891067/2a2187a2-f2de-4cf0-bf86-3c054dfef7fa)

**Comment by nobkd** [comment]
Have you checked if you have links in your content files that are referencing non-existent resources or missing pages in the content directory?
Sometimes that can lead to prerender errors...

**Reply by heiyehk** [reply]
I don't know, it's just that I deleted the [... slug ] file and it worked.
